src/GraphQL.Tests/Types/CustomScalarTests.cs

```diff
@@ -76,7 +75,7 @@ public void InOut_Literal(string field, string argumentValue, string expectedRes
         {
             var quotedArg = argumentValue == null ? "null" : $"\"{argumentValue}\"";
             var quotedResponse = expectedResponse == null ? "null" : $"\"{expectedResponse}\"";
-            var result = CreateQueryResult(responseType == Response.Success || responseType == Response.Error ? $"{{ \"{field}\":{quotedResponse} }}" : "null");
```

Comment:
"null" -> null ?

Reply:
The test didn't change; meaning, the query is returning the correct data both before and after these changes.  Somehow the code that created the test data for comparison changed.

There was only a single failure, where a non-null field threw a failure so `null` could not be returned in the `data` field, so `data` had to be `null`.  I'm still not exactly sure what changed, but probably some details of the replacement `ToDictionary` method used by tests.

The test case was quite confusing so I added comments and clarified the test cases better.  I also added code to explicitly create the `ExecutionResult` instance so that it is obvious what is set and why, with no reliance on `ToDictionary`.

Reply:
Done as indicated above

Reply:
Note 'NoField' is impossible; a proper response will always have a corresponding field in the map, or in case of error on a non-null field, it will bubble the error to the parent node.  So I renamed `ErrorNoField` to `ErrorDataNull`, which correctly represents the situation being tested here.

Reply:
ok